File: php/scripts/enterprise_schema.php
<?php
require_once(__DIR__."/../vendor/autoload.php");
require_once("utils.php");
use Namelivia\TravelPerk\ServiceProvider;
$isSandbox = false;
$travelperk = (new ServiceProvider())->build(getenv("API_KEY"), false);
$enterpriseSchema = $travelperk->scim()->discovery()->enterpriseUserSchema();
write_output("enterprise_schema", [
	$enterpriseSchema->id,
	$enterpriseSchema->name,
	$enterpriseSchema->description,
	count($enterpriseSchema->attributes),
	$enterpriseSchema->meta->resourceType,
	$enterpriseSchema->meta->location,
]);

File: python/scripts/enterprise_schema.py
import os
from travelperk_http_python.builder.builder import build
from utils import write_output
travelperk = build(os.environ["API_KEY"], False)
enterprise_schema = travelperk.scim().discovery().enterprise_user_schema()
write_output("enterprise_schema", [
    enterprise_schema["id"],
    enterprise_schema["name"],
    enterprise_schema["description"],
    enterprise_schema["attributes"],
    enterprise_schema["meta"]["resourceType"],
    enterprise_schema["meta"]["location"],
])
